python/desc/monitor/dbConnection.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from lsst.sims.catalogs.generation.db import DBObject
+
+class dbInterface(object):
+
+    def __init__(self):
```

Comment:
@jchiang87 we have been discussing this too! We would also like 
- to expose those parameters to the user 
- and thinking ahead to cases where the databases will not be read-only, avoid putting credential information in the source code.  @jbkalmbach has example snippets (based on other lsst codes) to do this
and obviously a PR would be a good place to discuss this more broadly within the group, and get further ideas.